benchmark_utils/common.py

```diff
@@ -51,13 +51,15 @@ def data_generator_sbibm(
     package.
 
     Args:
+    -----
         name: A task name (e.g. `'two_moons'`).
         train_size: The number of samples in the training set.
         test_size: The number of samples in the test set.
         ref_size: The number of reference posteriors.
         n_per_ref: The number of samples per reference posteriors.
 
     Returns:
+    --------
         A dictionary with training, test and reference sets as well as the parameters
         prior.
```

Comment:
can you stick to numpydoc style?

```suggestion
    Parameters
    ----------
    name: str
        A task name (e.g. `'two_moons'`).
    train_size: int
        The number of samples in the training set.
    test_size: int
        The number of samples in the test set.
    ref_size: int
        The number of reference posteriors.
    n_per_ref: int
        The number of samples per reference posteriors.

    Returns
    -------
    dict
        A dictionary with training, test and reference sets as well as the parameters
        prior.
```